pages/design-tokens/typesetting/letterspacing.md

```diff
@@ -120,7 +124,7 @@ Your context and coding style determine how you access USWDS letterspacing token
         </td>
         <td data-title="Example">
           <span>
-            $theme-heading-letterspacing: <code>-2</code>
+            $heading-letterspacing: <code>-2</code>
```

Comment:
> [!NOTE] 
> `$theme-heading-letterspacing` does not exist in USWDS. I could not find a letterspacing theme setting in USWDS, so I created a sample setting that looks less like a USWDS theme setting. Alternatively, we could remove the setting example entirely or maybe there is a theme setting that would be a good fit that I missed. Curious to hear your thoughts.

Reply:
I'm slightly worried about users thinking a demo variable actually exists in the design system. I wonder if there's a naming convention we could use to help get the idea that it's theoretical?

```scss
$letterspacing-example: -2
```

I might be overthinking it.

I do think that removing the `$theme-` prefix is a good idea.

Reply:
I think you make a really good point here. It feels risky to use settings in our docs that don't exist in USWDS, even if we can find a more intuitive naming convention. 

Since we don't have a theme setting that uses letterspacing tokens (As far as I can tell), I removed the settings example entirely from the "Using letterspacing tokens" table in [c18ce60](https://github.com/uswds/uswds-site/pull/2379/commits/c18ce6099c8b14e6e5f2d7821c9ca090afbad43c). Let me know if you see any issues with that!